src/styles/product_card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  $cardPrefix: &;
+  width: 200px;
+  padding: 30px 0 15px;
+  flex-wrap: wrap;
+  justify-content: center;
+  font-size: 11px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 0 45px;
+  }
+
+  &__info {
+    width: 170px;
+    flex-wrap: wrap;
```

Comment:
use flex-direction instead of this -> this will allow you to remove width 100% everywhere